### PR TITLE
open in editor button for yellow box

### PR DESCRIPTION
**Pull request by skv-headless**
Here is the demo https://www.dropbox.com/s/ljh7f9654sqgdqc/demo.mov?dl=0

### CODE REVIEW

**Comment by facebook-github-bot-1** [comment]
By analyzing the blame information on this pull request, we identified @yungsters and @davidaurelio to be potential reviewers.

File: Libraries/ReactIOS/YellowBox.js [hunk 1]
```diff
@@ -172,12 +174,23 @@ const WarningRow = ({count, warning, onPress}) => {
...
   const fileName = fileParts[fileParts.length - 1];
+
   return (
-    <Text style={styles.inspectorCountText}>
-      {`${fileName}:${frame.lineNumber}`}
-    </Text>
+    <TouchableOpacity
+      style={styles.openInEditorButton}
+      onPress={() => {
+        fetch(getDevServer().url + 'open-stack-frame', {
```
Comment (frantic) [inline Libraries/ReactIOS/YellowBox.js]: please extract this to a separate `openFileInEditor` function (element inspector has the same code)

**Comment by frantic** [comment]
Great stuff, thank you! From design standpoint, would be cool to somehow indicate that the rows are "tappable" (not required for this PR though, just an idea)

**Comment by skv-headless** [comment]
@frantic done. I hope ```openFileInEditor.js``` in a write place. Since you fixed scroll in yellow box in next PR I can make similar stack trace as in red box.

**Comment by frantic** [comment]
Thanks for the quick turnaround. This looks good!

**Comment by frantic** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-0** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1558034931167722/int_phab) to review.

**Comment by facebook-github-bot-4** [comment]
I tried to merge this pull request into the Facebook internal repo but some checks failed. To unblock yourself please check the following: Does this pull request pass all open source tests on GitHub? If not please fix those. Does the code still apply cleanly on top of GitHub master? If not can please rebase. In all other cases this means some internal test failed, for example a part of a fb app won't work with this pull request. I've added the Import Failed label to this pull request so it is easy for someone at fb to find the pull request and check what failed. If you don't see anyone comment in a few days feel free to comment mentioning one of the core contributors to the project so they get a notification.

**Comment by skv-headless** [comment]
@frantic do I need to do something? As I understood test fails unrelated

**Comment by frantic** [comment]
Sorry for the slowness, our Android end-to-end tests fail with cryptic errors, I'll look into this.

**Comment by skv-headless** [comment]
I'm learning how tests works. I think I will be able to fix by myself approximately in a month 😄

**Pull request closed without merging**